src/components/views/elements/MessageEditor.js

```diff
@@ -116,6 +116,10 @@ export default class MessageEditor extends React.Component {
                     autoComplete.onUpArrow(event); break;
                 case "ArrowDown":
                     autoComplete.onDownArrow(event); break;
+                case "ArrowLeft":
```

Comment:
No need to add left/right arrow keys for the new composer yet, I need to test if it doesn't interfere with caret navigation. Let's just do enough here so it isn't broken.

Reply:
Roger that, removed.